client/helpers.go

```diff
@@ -216,14 +245,14 @@ func addKeyForRole(kdb *keys.KeyDB, role string, key data.PublicKey) error {
 // signs and serializes the metadata for a canonical role in a tuf repo to JSON
 func serializeCanonicalRole(tufRepo *tuf.Repo, role string) (out []byte, err error) {
 	var s *data.Signed
-	switch role {
-	case data.CanonicalRootRole:
+	if role == data.CanonicalRootRole {
```

Comment:
I tried that first, but apparently `case` doesn't seem to work with `_, ok := tufRepo.Targets[role]; ok` - I kept getting syntax errors :|